gateway-server/src/main/java/org/apache/knox/gateway/util/KnoxCLI.java

```diff
@@ -174,7 +175,7 @@ GatewayServices getGatewayServices() {
     return services;
   }
 
-  private void initializeServices(boolean persisting) throws ServiceLifecycleException {
+  private void initializeServices(boolean persisting) throws ServiceLifecycleException, GatewayConfigurationException {
     GatewayConfig config = getGatewayConfig();
```

Comment:
Does the KnoxCLI really need to fail here if the server's Kerberos config is not correct?

Reply:
Fixed